src/pages/Portfolio/MyKiva/MyKivaPage.vue

```diff
@@ -323,10 +377,8 @@ watch(isAchievementDataLoaded, () => {
 		nextTick(() => {
 			// Scroll to section once async data is loaded
 			const targetId = window?.location?.hash?.replace('#', '');
-			const targetElement = document?.getElementById(targetId);
-			if (targetElement) {
-				targetElement.scrollIntoView({ behavior: 'smooth' });
-			}
+			scrollToTarget(targetId);
+			checkGuestAchievementsToScroll();
```

Comment:
What happens here when scroll gets called twice for a guest?

Reply:
We are not passing hash for guest cases, but I updated this to ensure scroll if hash is passed.